README.md

```diff
@@ -56,7 +56,9 @@ chaos_monkey:
         kill_app:
             active: false
     watchers: # currently watchers can be enabled/disabled only in container compile time
-        request: true
+        request:
+            enabled: true
```

Comment:
for consistency, let's rename this option to `active`

Reply:
ok,I read how this option (`canBeDisabled`) works and it can stay as is. Thanks for your contribution